cypress/plugins/docker/index.ts

```diff
@@ -36,12 +36,22 @@ export async function dockerRun(opts: {
     const params = opts.params ?? [];
 
     if (params?.includes("-v") && userInfo.uid >= 0) {
-        // On *nix we run the docker container as our uid:gid otherwise cleaning it up its media_store can be difficult
-        params.push("-u", `${userInfo.uid}:${userInfo.gid}`);
-
         if (await isPodman()) {
-            // keep the user ID if the docker command is actually podman
-            params.push("--userns=keep-id");
+            // Note: this setup is for podman rootless containers.
+
+            // In podman, run as root in the container, so we're the current
+            // user on the host. This is probably the default since Synapse's
+            // Dockerfile doesn't specify, but we're being explicit here
+            // because it's important for the permissions to work.
+            params.push("-u", "0:0");
+
+            // Tell Synapse not to switch UID
+            params.push("-e", "UID=0");
+            params.push("-e", "GID=0");
+        } else {
+            // On *nix we run the docker container as our uid:gid - otherwise,
+            // cleaning it up its media_store can be difficult
```

Comment:
I think this comment is relevant to both podman and docker, though the means of achieving it is different between the two.

Suggest moving out of the `if/else` condition.

Reply:
Done